app/views/shared/_accordion.html.erb

```diff
@@ -4,24 +4,26 @@
 
     <div class="card">
     <div class="card-header d-flex justify-content-between" id="heading<%=index.humanize.capitalize%>">
+    <div class="flex-grow-1">
```

Comment:
why do we have 3 levels of divs instead of only one div with all the classes and ids?

Reply:
Solved